platform/docs/docs/platform/extensions/modules/hpModule.md

```diff
@@ -246,7 +246,23 @@ A list of criteria for the protocol along with the provided points for ranking.
 
   - `constraint`: the constraint that needs to be satisfied for the attribute. It accepts a `validator` which can be
     [`equals`, `doesNotEqual`, `contains`, `doesNotContain`, `startsWith`, `endsWidth`]
-
+   
+  - | Rule 	| Single Value 	| Array Value 	| Example 	|
+    |---	|---	|---	|---	|
+    | equals 	| === 	| All members are === in same order 	| value = ['abc', 'def', 'GHI']<br>testValue = 'abc' (Fail)<br><br>          = ['abc'] (Fail)<br><br>          = ['abc', 'def', 'GHI'] (Valid)<br><br>          = ['abc', 'GHI', 'def'] (Fail)<br><br>          = ['abc', 'def'] (Fail)<br><br>value = 'Attenuation Corrected'<br>testValue = 'Attenuation Corrected' (Valid)<br>          = 'Attenuation' (Fail)<br> 	|
```

Comment:
I believe the tests are failing on how the `<br>` tag is used - I suspect you need to use <br /> as a closed tag value (XHTML syntax).  Try building hte docs locally to test.